src/ui/public/chrome/services/breadcrumb_state.d.ts

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
Because I didn't want to descend into having to type all of the angular stuff, but I think I also did this before I discovered `@ts-ignore`. I can probably move this to TS using that.